plugins/islandora_context_reaction_restrict_by_ip.inc

```diff
@@ -111,7 +111,7 @@ class IslandoraContextReactionRestrictByIp extends context_reaction {
       if (!strlen($range)) {
         continue;
       }
-      list($low, $high) = explode(':', $range);
+      list($low, $high) = array_pad(explode(':', $range, 2), 2, NULL);
```

Comment:
@mjordan Correct - when I wrote "using the IP range notation even for single IP addresses" I meant using notation like:

``123.45.12.234:123.45.12.234``